templates/components/products/options/product-list.html

```diff
@@ -25,7 +25,11 @@
                 <li class="productOptions-list-item" data-product-attribute-value="{{id}}">
                     {{#if image}}
                         <figure class="productOptions-list-item-figure">
-                            <img class="productOptions-list-item-image lazyload" data-sizes="auto" src="{{cdn 'img/loading.svg'}}" data-src="{{getImage image 'productthumb_size'}}" alt="{{image.alt}}" title="{{image.alt}}">
+                            {{> components/common/responsive-img
+                                image=image
+                                class="productOptions-list-item-image"
+                                lazyload='lazyload+lqip'
```

Comment:
I'm forcing lazyloading on these because the entire structure of the product list element gets significantly modified by JS after the page loads, so there's no need to load these immediately - it's better to wait for the sizing to settle so the browser can pick the right size.